docs/ENV.md

```diff
@@ -14,38 +14,17 @@ You can create a 32-digit secret key from this website: https://generate.plus/en
 If true, it enables scheduled refreshes. Can also be set to a number representing how many seconds to wait before triggering another refresh. Recommended value is true for production deployments. Defaults to false.
 
 ### `PLIO_ORGANIZATIONS_SCHEMAS`
```

Comment:
I had made my earlier suggestions for heading based on this - saw the file now and realized that the mistake is here.
```suggestion
#### `PLIO_ORGANIZATIONS_SCHEMAS`
```